### DISCUSSION TITLE
Where to get latest version without manually building?

### DISCUSSION THREAD

**Discussion by atirutw** [fnuecke/oc2#140] [Q&A] [answered]
The build on [OpenComputers build server](https://ci.cil.li/job/oc2-1.16-forge/) is very outdated

**Comment by sannelo** [comment, marked as answer]
https://github.com/fnuecke/oc2/releases

**Comment by fnuecke** [comment]
The releases also get published to [curseforge](https://www.curseforge.com/minecraft/mc-mods/oc2/files) if that's more convenient for you.
